File: programmers/Lv1/Lv1_같은_숫자는_싫어.py
def solution(arr):
    temp = arr[0]
    answer = []
    answer.append(temp)
    for i in arr:
        if temp != i:
            answer.append(i)
        temp = i
    return answer

# print(solution([1,1,3,3,0,1,1]))
# print(solution([4,4,4,3,3]))

File: programmers/Lv1/Lv1_같은_숫자는_싫어.js
// 자바스크립트를 써야될 일이 있어서 자바스크립트 연습겸 알고리즘 문제 풀어보기
function solution(arr)
{
    var answer = [];
    
    for (let i=0; i<arr.length; i++ ){
        if (arr[i] !== arr[i+1]) {
            answer.push(arr[i]);
        }
    }
    return answer;
}
console.log(solution([1,1,3,3,0,1,1])) // [1,3,0,1]
console.log(solution([4,4,4,3,3])) // [4,3]